Pipfile.lock

```diff
@@ -782,6 +805,14 @@
             "index": "pypi",
             "version": "==1.4.3"
         },
+        "appnope": {
```

Comment:
Where is appnope being used?

Reply:
I think this is being grabbed by pipenv install. It keep popping up. Maybe we should just leave it in..